.github/dependabot.yml

```diff
@@ -6,7 +6,11 @@ updates:
     schedule:
       # Check for updates to Go modules every weekday
       interval: "daily"
-
+    groups:
+      # Group all terraform-plugin-(go|sdk|framework|testing) dependencies together
+      "terraform-plugin":
+        patterns:
+          - "github.com/hashicorp/terraform-plugin-*"
```

Comment:
This repo doesn't use any of these Go modules
```suggestion
```

Reply:
good call! done